utils/mgmt_system/openstack.py

```diff
@@ -621,3 +639,27 @@ def delete_template(self, template_name):
         template = self._find_template_by_name(template_name)
         template.delete()
         wait_for(lambda: not self.does_template_exist(template_name), num_sec=120, delay=10)
+
+    def stack_get(self, stack_id):
+        return self.stackapi.stacks.get(stack_id)
+
+    def stack_exist(self, stack_name):
+        stacks = [stack for stack in self.stack_get(stack_name)
+            if stack.stack_name == stack_name]
+        if (len(stacks) >= 1):
```

Comment:
Pythonism: Any sequence with length >= 1 is considered "True", so there are a couple of ways to spell this conditional that are more "pythonic":

`return True if stacks else False`
```
if stacks:
  return True
else:
  return False
```

`return bool(stacks)`
(my favorite)